src/sentry/static/sentry/app/components/forms/multiSelectField.jsx

```diff
@@ -27,20 +34,13 @@ export default class MultiSelectField extends FormField {
     this.setValue(value);
   };
 
-  renderArrow = () => {
-    return <span className="icon-arrow-down" />;
-  };
-
   getField() {
     return (
-      <StyledSelect
+      <MultiSelectControl
         id={this.getId()}
-        onChange={this.onChange}
         value={this.state.value}
-        multi={true}
-        arrowRenderer={this.renderArrow}
-        style={{width: 200, overflow: 'visible'}}
         {...this.props}
+        onChange={this.onChange}
```

Comment:
We don't want user to override `onChange` because we want to always keep behavior of `FormField`. If you do not need `FormField` behavior, then use a `Control`.